### ISSUE TITLE
[MU4 Issue] Repeat last measure symbol has no effect on playback

### ISSUE THREAD

**Issue by rgreen5** [musescore/MuseScore#12400]
**Describe the bug**

The repeat last measure symbol has no effect on playback.

**To Reproduce**

1. Open attached score. 
[repeat_symbol.mscz.zip](https://github.com/musescore/MuseScore/files/9111827/repeat_symbol.mscz.zip)

2. Play.

RESULT: There is no sound as the playback cursor passes the measure with the repeat symbol in it.

**Expected behavior**

Repeat last measure symbol should affect playback.

**Platform information**
OS: Linux Mint 20.1, Arch.: x86_64, MuseScore version (64-bit): 4.0.0-2667920094, revision: github-musescore-musescore-0f1e7fb

**Comment by Tantacrul** [comment]
Yes, this is a repeat of https://github.com/musescore/MuseScore/issues/10922

We consider it to be a high priority.
